tests/nrf/src/bin/wifi_esp_hosted_perf.rs

```diff
@@ -23,6 +21,10 @@ bind_interrupts!(struct Irqs {
     RNG => embassy_nrf::rng::InterruptHandler<peripherals::RNG>;
 });
 
+// Test-only wifi network, no internet access!
```

Comment:
Would it be possible to pick these up from ENV variables compile-time [core](https://doc.rust-lang.org/core/index.html)::[env](https://doc.rust-lang.org/core/macro.env.html#) instead of hardcoding these?